ui/server/develConf/demo/application.conf

```diff
@@ -0,0 +1,85 @@
+# copy-paste of demo-application.conf
```

Comment:
For reproducing problems associated with demo configcreator (in example module). Perfect solution would be using placeholders in demo-application.conf, because some little changes in this conf was needed. But I would like to start with copy-paste and if it will has been used too often, move to one config with some placeholders. What do you think about it?

Reply:
maybe let's add comment in file, so we don't forget :)

Reply:
ok